clients/src/main/java/org/apache/kafka/common/metrics/Sensor.java

```diff
@@ -41,7 +41,34 @@
     private volatile long lastRecordTime;
     private final long inactiveSensorExpirationTimeMs;
 
-    Sensor(Metrics registry, String name, Sensor[] parents, MetricConfig config, Time time, long inactiveSensorExpirationTimeSeconds) {
+    public enum RecordLevel {
+        SENSOR_INFO, SENSOR_DEBUG;
+        public static final String SENSOR_INFO_STR = "INFO";
+        public static final String SENSOR_DEBUG_STR = "DEBUG";
+
+        public static RecordLevel fromString(String recordLevel) {
+            if (recordLevel.equals(SENSOR_INFO_STR)) {
+                return SENSOR_INFO;
+            } else if (recordLevel.equals(SENSOR_DEBUG_STR)) {
+                return SENSOR_DEBUG;
+            } else throw new IllegalArgumentException("record Level " + recordLevel + " type not defined");
+        }
+
+        public static String toString(RecordLevel recordLevel) {
+            if (recordLevel == SENSOR_DEBUG) {
+                return SENSOR_DEBUG_STR;
+            } else if (recordLevel == SENSOR_INFO) {
+                return SENSOR_INFO_STR;
+            }
```

Comment:
Instead of doing this, we can simply override `toString` in each enum. However, if we go with my suggestion of renaming the enum values, the `toString` will be the right one by default, I think.